### ISSUE TITLE
Return keyword is removed when it should not.

### ISSUE THREAD

**Issue by RabugenTom** [nicklockwood/SwiftFormat#104]
Hi Nick,
I've found an edge case: with v0.25.1, 
````swift
var foo: T? {
    if let bar = try? f as? T {
        return bar
    }
   return nil
}
````
is formatted as
````swift
var foo: T? {
    if let bar = try? f as? T {
        bar
    }
    return nil
}
````
SwiftFormat thinks it is in some closure and removes the `return` keyword. Please note that it needs the `try?` (or `try`) and the `as? T` cast to happen. `return nil` is only here for cosmetics. The minimal failing code sample is:
````swift
var foo: T? {
    if let bar = try f as? T {
        return bar
    }
}
````

**Comment by nicklockwood** [comment]
@RabugenTom I believe this was fixed in version 0.25.2

**Comment by RabugenTom** [comment]
My bad, I thought I had the latest version.
This is indeed fixed with version 0.25.2.
Thanks!
